src/torrent.cpp

```diff
@@ -10613,15 +10614,17 @@ namespace libtorrent
 			, std::bind(&torrent::on_piece_verified, shared_from_this(), _1)
 			, reinterpret_cast<void*>(1));
 	}
-
-	announce_entry* torrent::find_tracker(tracker_request const& r)
```

Comment:
since this version of ``find_tracker()`` only looks at the URL (and not the type), it seems like it's misleading. Would you mind removing it? and have callers just use your new version instead.